client/lclusterc.go

```diff
@@ -0,0 +1,324 @@
+
+    // Input validation
+    if len(uuid) < 1 {
+        fmt.Printf("removeJobFromServer() --> invalid input")
+        return
+    }
+
+    // attempt to cast the pid into an int since that is StopJobRequest
+    // uses as the Pid var type
+    pid, err := strconv.ParseInt(uuid, 10, 64)
+
+    // if an error occurs, print it out
+    if err != nil || pid < 1 {
+        fmt.Printf("Please enter a valid int64 value.\n")
+        return
+    }
+
+    // Dial a connection to the grpc server.
+    connection, err := grpc.Dial(grpcServerAddr + grpcPort,
+      grpc.WithInsecure())
+
+    // Safety check, ensure no errors have occurred.
+    if err != nil {
+        fmt.Printf("removeJobFromServer() --> unable to connect to grpc server at address")
+        return
+    }
+
+    // Defer the connection for the time being, but eventually it will be
+    // closed once we've finished with it.
+    defer connection.Close()
+
+    // Create an lcluster client
+    lcluster_client := pb.NewLclusterdClient(connection)
+
+    // Assemble a stop job request object
+    request := &pb.StopJobRequest{ Pid: pid }
+
+    // Grab the current background context.
+    ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+
+    // Using the stop job request defined above, go ahead and attempt to
+    // stop the job
+    response, err := lcluster_client.StopJob(ctx, request)
+
+    // Cancel the current context since this has either generated a
+    // response or an error.
+    cancel()
+
+    // Safety check, ensure that no error has occurred.
+    if err != nil {
+        fmt.Printf("removeJobFromServer() --> the following error has " +
+                   "occurred...\n")
+        fmt.Printf(err.Error())
+        return
+    }
+
+    // check the return code from the StopJobResponse object, if it -1 then
+    // an error has occurred.
+    if response.Rc == SJR_FAILURE {
+        fmt.Printf("Server has responded with the following error:\n")
+        fmt.Printf(response.Error + "\n")
+        return
+    }
+
+    // with a return code of `1` then the process could not be found, but
+    // the server otherwise experienced no error while processing the
+    // request
+    if response.Rc == SJR_DOES_NOT_EXIST {
+        fmt.Printf("No such process exists with Uuid: " +
+               strconv.FormatInt(pid, 10) + "\n")
+        return
+    }
+
+    // Since this has obtained a response, go ahead and return the result
+    if response.Rc == SJR_SUCCESS {
+        fmt.Printf("Successfully removed job of pid: " +
+                   strconv.FormatInt(pid, 10) + "\n")
+        return
+    }
+
+    // otherwise if none of the above happened, print out a message
+    // stating this is unknown, with the response code
+    fmt.Printf("The following undefined response code was given back: " +
+      strconv.FormatInt(response.Rc,10) + "\n")
+    fmt.Printf("Consider contacting the server operator.\n")
+}
+
+//
+// Main
+//
+func main() {
```

Comment:
I have since done so for the both the server main.go and the client.

https://github.com/rbisewski/lclusterd/pull/2/commits/557528b487d9e97c4ce1aa2dd455301ec4312ae0

https://github.com/rbisewski/lclusterd/pull/2/commits/302fce7752fa517a5c9ec8c2a108a45d817a135d